### PR TITLE
[OSDOCS-7261] CPMS migration procedure for RHOSP

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.14

Issue: [OSDOCS-7261](https://issues.redhat.com/browse/OSDOCS-7261)

Link to docs preview: 
- https://65266--docspreview.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-managing#nodes-control-plane-osp-migrating_nodes-nodes-managing
- https://65266--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-using#cpmso-using
- https://65266--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-user-kuryr#installation-user-infra-generate-k8s-manifest-ignition_installing-openstack-user-kuryr

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by maxwelldb** [comment]
@EmilienM Thoughts on the state of this PR for OSDOCS-7261?

File: modules/cpmso-feat-replace.adoc [hunk 1]
```diff
@@ -8,6 +8,17 @@
 
 To replace a control plane machine in a cluster that has a control plane machine set, you delete the machine manually. The control plane machine set replaces the deleted machine with one using the specification in the control plane machine set custom resource (CR).
 
+.Prerequisites
+
+* If your cluster runs on {rh-openstack-first}, you must disable the {rh-openstack} compute node that the machine runs on by running the following command:
```
Comment (maxwelldb) [inline modules/cpmso-feat-replace.adoc:13]:
```suggestion
* If your cluster runs on {rh-openstack-first} and you need to evacuate a compute server, such as for an upgrade, you must disable the {rh-openstack} compute node that the machine runs on by running the following command:
```

File: modules/cpmso-feat-replace.adoc [hunk 2]
```diff
@@ -8,6 +8,17 @@
...
++
+[source,terminal]
+----
+$ openstack compute service set <target_node_host_name> nova-compute --disable
```
[the remark this answers is not in the document]
Comment (maxwelldb) [reply modules/cpmso-feat-replace.adoc:17]: Does https://github.com/openshift/openshift-docs/pull/65266/files#r1358227576 suffice? @EmilienM

[the remark this answers is not in the document]
Comment (EmilienM) [reply modules/cpmso-feat-replace.adoc:17]: yes

**Comment by maxwelldb** [comment]
Thanks!

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**